iina/QuickSettingViewController.swift

```diff
@@ -394,23 +404,19 @@ class QuickSettingViewController: NSViewController, NSTableViewDataSource, NSTab
     }
```

Comment:
We might want to implement "snapping" instead of `allowsTickMarkValuesOnly`–but that would be a different pull request.